plugins/UsersManager/API.php

```diff
@@ -538,10 +538,23 @@ private function enrichUser($user)
         if (!empty($user)) {
             unset($user['token_auth']);
         }
+        $lastSeen = $this->addLastSeen($user);
+
+        if ($lastSeen) {
+          $user['last_seen'] = $lastSeen;
```

Comment:
Maybe it would be better to always set the `last_seen`, so the returned data has a consistent format. We could set it to `0` or `false`  if it would be empty